lib/dialects/sqlite/query.js

```diff
@@ -24,6 +26,28 @@ Query.prototype.getInsertIdField = function() {
   return 'lastID';
 };
 
+Query.prototype.$collectModels = function(include, prefix) {
```

Comment:
I have a feeling this function does a lot of what `groupJoinData` does - but there's no way I'm looking into that function without being forced to :D

Basically, it creates a map from include alisesses to models. Fx `'users.projects'` -> project model